inc/head/header-css.inc

```diff
@@ -1,9 +1,9 @@
 <link rel='stylesheet' id='tna-google-fonts-css'
       href='https://fonts.googleapis.com/css?family=Open+Sans%3A400%2C700%2C400italic%2C700italic%7CBitter&#038;ver=4.4.11'
       type='text/css' media='all'/>
-<link rel='stylesheet' href="css/base-sass.min.css"/>
-<link rel='stylesheet' href="css/new-styles.css"/>
+<link rel='stylesheet' href="<?php echo SITE_URL ?>css/base-sass.min.css"/>
+<link rel='stylesheet' href="<?php echo SITE_URL ?>css/new-styles.css"/>
```

Comment:
There's no need for new-style.css as we do have one main css file which is base-sass.min.css minified file. All the css it needs to be written inside base-sass.scss with the grunt comand running in the background(console)